### DISCUSSION TITLE
parallel but different types of stimuli

### DISCUSSION THREAD

**Discussion by nourezzeddine** [jspsych/jsPsych#1372] [Q&A] [answered]
Hello,
I am trying to do a comprehension task, where participants look at different images and listen to an audio stimulus simultaneously.
I've tried creating the experiment with opensesame using coroutine function then run it with jatos. This doesn't work because coroutine function is not supported by jatos. 
**I wanted to know if that is possible with jspsych 'presenting a text / multiple images with an audio sound siultaneously' before I start learning how to use jspsych.** 
I have no background in coding.

Thank you  
Nour

**Comment by jodeleeuw** [comment, marked as answer]
Hi @nourezzeddine ,

Yes, you can use the `audio-keyboard-response` plugin to do this. A sample trial with text and images might look like this:

```js
var text_and_audio = {
	type: 'audio-keyboard-response',
	stimulus: 'my-audio-file.mp3',
	prompt: '<img src="my-image.png"><p>Some text</p>'
}
```

**Reply by nourezzeddine** [reply]
Thank you Jodeleeuw :)
